configure.ac

```diff
@@ -1197,11 +1197,8 @@ AC_ARG_WITH([systemdsystemunitdir],
 		;;
 	esac
 ], [])
-dnl Override installation directory, with the local installation
```

Comment:
I am not sure removal of the comment is proper: we do still override the value if set, can keep the explanation why...

Reply:
That comment referred to code removed by f65dc913d862 ("Undo ${systemdsystemunitdir} mangling.").